### ISSUE TITLE
[css-variables] Event when property value changes

### ISSUE THREAD

**Issue by ByteEater-pl** [w3c/csswg-drafts#5106]
It would be useful to have an event dispatched to a (pseudo)element when its CSS property (any, actually, but custom ones seem to be a good first step) changes.

[Relevant section](https://drafts.csswg.org/css-variables/#apis)

**Comment by tabatkins** [comment]
Agreed that this would be useful, but it's likely part of a larger effort; at minimum, it's more than level 1 of this spec wants to handle. I'll punt this to level 2. It might go somewhere else eventually, but that's at least a good place to park it.
